src/remote.js

```diff
@@ -259,20 +278,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
```

Comment:
The last error catch is executed when the server cannot process a request (wrong URL, server down, ect). In that case, it will return something like `{"status":"fail","code":404,"message":"Not found"}`, which will be part of the error message. It seems clear for me but maybe we should add something like `Cannot check the build availability, server response: ..... `. What do you think @masahirotanaka ?